HHS/Head-Start-TTADP
# [Prod] Display which program a CFO belongs to on the leadership widget

## Description of change

Display the program type for *all* leadership on the recipient profile/leadership widget.

## How to test
- Visit an RTR profile in staging - for example, [this one](https://tta-smarthub-staging.app.cloud.gov/recipient-tta-records/1317/region/1/profile)
- You will see that the CFO now says "CFO for early head start" or "CFO for head start" depending on the associated program

## Issue(s)

* https://ocio-jira.acf.hhs.gov/browse/TTAHUB-2300


## Checklists

### Every PR



### Before merge to main


### Production Deploy


### After merge/deploy
